core/client/fs/src/main/java/alluxio/client/file/cache/store/LocalPageStore.java

```diff
@@ -69,12 +69,14 @@ public void put(PageId pageId, byte[] page) throws IOException {
   }
 
   @Override
-  public ReadableByteChannel get(PageId pageId) throws IOException, PageNotFoundException {
+  public ReadableByteChannel get(PageId pageId, int pageOffset)
```

Comment:
Precondition to check page Offset is positive?

Reply:
Added.